src/azure-cli/azure/cli/command_modules/managedservices/_help.py

```diff
@@ -0,0 +1,174 @@
+          short-summary: |
+             Optional: The plan publisher.
+        - name: --plan-version
+          short-summary: |
+             Optional: The plan version.
+        - name: --definition-id
+          short-summary: |
+             Optional:  Can be used to override the generated registration definition id.
+    examples:
+        - name: Creates a registration definition under the default subscription scope with the required parameters.
+          text: az managedservices definition create --name mydef --tenant-id dab3375b-6197-4a15-a44b-16c41faa91d7 --principal-id b6f6c88a-5b7a-455e-ba40-ce146d4d3671 --role-definition-id ccdd72a7-3385-48ef-bd42-f606fba81ae7
+        - name: Creates a registration definition under the given subscription scope with the required parameters.
+          text: az managedservices definition create --name mydef --tenant-id dab3375b-6197-4a15-a44b-16c41faa91d7 --principal-id b6f6c88a-5b7a-455e-ba40-ce146d4d3671 --role-definition-id ccdd72a7-3385-48ef-bd42-f606fba81ae7 --subscription e716db1e-393d-4f2a-81fe-2577e92c9200
+"""
+
+helps['managedservices definition show'] = """
+    type: command
+    short-summary: Gets a registration definition.
+    long-summary: Gets a registration definition when given its identifier or the fully qualified resource id. When resource id is provided, the subscription paramter is ignored.
+    parameters:
+        - name: --definition
+          short-summary: |
+             The identifier (guid) or the fully qualified resource id of the registration definition. When resource id is used, subscription id and resource group parameters are ignored.
+    examples:
+        - name: Gets the registration definition given its identifier under the default subscription scope.
+          text: az managedservices definition show --definition af8772a0-fd9c-4ddc-8ad0-7d4b3913d7dd
+        - name: Gets the registration definition given its identifier and subscription id.
+          text: az managedservices definition show --definition af8772a0-fd9c-4ddc-8ad0-7d4b3913d7dd --subscription 39033314-9b39-4c7b-84fd-0e26e55f15dc
+        - name: Gets the registration definition given its fully qualified resource id.
+          text: az managedservices definition show --definition /subscriptions/39033314-9b39-4c7b-84fd-0e26e55f15dc/providers/Microsoft.ManagedServices/registrationDefinitions/1d693e4f-9e79-433a-b3a2-afce1f8b61ec
+"""
+
+helps['managedservices definition delete'] = """
+    type: command
+    short-summary: Deletes a registration definition.
+    long-summary: Deletes a registration definition when given its identifier or the fully qualified resource id. When resource id is provided, the subscription paramter is ignored.
+    parameters:
+        - name: --definition
+          short-summary: |
+             The identifier (guid) or the fully qualified resource id of the registration definition. When resource id is used, subscription id and resource group parameters are ignored.
```

Comment:
There is a lot of duplication here that could make maintenance difficult or at least irritating. Typically, these short summaries would simply be registered at the appropriate scope in params.py to avoid this duplication. Use of help.py for parameters is only necessary in the event long summaries or populator commands are needed (features not available in params.py).